settings_tab.py

```
@@ -0,0 +1,361 @@
+
+        ttk.Separator(parent, orient="horizontal") \
+            .grid(column=0, row=row, columnspan=3, sticky="ew", padx=10, pady=10)
+        row += 1
+
+        # --- Font family ---
+        ttk.Label(parent, text="Font family:") \
+            .grid(column=0, row=row, sticky="w", padx=10, pady=4)
+        families = ui_theme.list_font_families(self.root)
+        self.font_family_combo = ttk.Combobox(
+            parent, textvariable=self.font_family_var,
+            values=[""] + families, width=30,
```

Ruff (0.15.10)

[warning] Consider `["", *families]` instead of concatenation

Replace with `["", *families]`

(RUF005)